### ISSUE TITLE
Unable to clear alt text, video and image elements

### ISSUE THREAD

**Issue by spacedmonkey** [GoogleForCreators/web-stories-wp#1128]
## Bug Description
I am unable to clear the alt / title accessibility fields on the video element. When I empty the field, either by using the delete key or cross, it refills with default value again. 

## Expected Behaviour
User should be able to clear field. 

## Steps to Reproduce

See video. 

## Screenshots

https://youtu.be/DEq9MiOyjI4

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by spacedmonkey** [comment]
I believe was broken here https://github.com/google/web-stories-wp/commit/b05d3148e5c3d5d666ccf177204ff9ad88ef25f8#diff-04665789db36c73169431fd3a792dfd0R44-R46
